src/js/utils/random-id-generator.js

```diff
@@ -0,0 +1,20 @@
+export const feedShownIdLength = 12;
```

Comment:
Constants should be in uppercase with underscore word separators `FEED_SHOWN_ID_LENGTH`.